fmriprep/workflows/bold/confounds.py

```diff
@@ -387,6 +396,8 @@ def _pick_wm(files):
         (wm_tfm, wm_msk, [('output_image', 'roi_file')]),
         (wm_msk, mrg_lbl, [('out', 'in2')]),
         (inputnode, mrg_lbl, [('bold_mask', 'in3')]),
+        (acc_msk, mrg_lbl, [('out', 'in4')]),
+        (tcc_msk, mrg_lbl, [('out', 'in5')]),
```

Comment:
Don't we want to inspect the `high_variance_masks` output of `tcompcor`, rather than just the input mask?

Reply:
probably yes, I'll check

Reply:
okay yes. the correlation of the mask currently connected will be always near 1.0. Just added a commit.